### PR TITLE
An instance to connect to with JumpCloud.

### PR DESCRIPTION
**Pull request by xschildw**
Just want to provision a nano for people to connect to after setting JumpCloud up. Will delete when we're all up and running.

### CODE REVIEW

File: config/prod/test-jumpcloudclientsetup.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Provision an EC2 instance
+template_path: managed-ec2-v4.yaml
```
Comment (zaro0508) [inline config/prod/test-jumpcloudclientsetup.yaml]: please update, there is a new version. managed-ec2-v5.yaml

File: config/prod/test-jumpcloudclientsetup.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+stack_name: test-jumpcloudclientsetup
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "IT"
```
Comment (zaro0508) [inline config/prod/test-jumpcloudclientsetup.yaml]: shouldn't this be "Platform"?

File: config/prod/test-jumpcloudclientsetup.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "IT Infrastructure"
+  # The resource owner
+  OwnerEmail: "x.schildwachter@sagebase.org"
+  # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "t3.nano"
+  # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
+  VolumeSize: "8"
```
Comment (zaro0508) [inline config/prod/test-jumpcloudclientsetup.yaml]: you don't need to specify this.  it's already the default

**Comment by xschildw** [comment]
Good points. Closing this PR and creating in SandBox.

**Pull request closed without merging**